narwhals/_arrow/expr.py

```diff
@@ -245,6 +245,9 @@ def n_unique(self: Self) -> Self:
     def std(self: Self, ddof: int = 1) -> Self:
         return reuse_series_implementation(self, "std", ddof=ddof, returns_scalar=True)
 
+    def var(self: Self, ddof: int = 1) -> Self:
```

Comment:
In here and as well as all other compliant Series/Expr implementation we are now trying to avoid providing default values, could you adjust as:

```suggestion
    def var(self: Self, ddof: int) -> Self:
```

Reply:
Just one question here. Pandas pyarrow, and polars handles variance calculation using `ddof` differently in different versions. How do I handle it?

Reply:
I imagine you are referring to the failing test. Not sure how `std` doesn't have such problem.
Or are there other instances of this happening?

Reply:
Yes, I'm referring to the failing tests. 
For instance, to this `data = {"a": [1, 1, 1, 2], "b": [1, None, 2, 3]}`,
when I run `nox`, 
- Python 3.8 expects `{"a": [1, 2], "b": [0, None]}`, 
- Python >= 3.9 expects `{"a": [1, 2], "b": [0.5, None]}`.

Interestingly `std` does not have this issue!

Reply:
I will take a deeper look later today! Thanks for pointing this out :)

Reply:
It seems to be an issue with pyarrow backend and pandas v2.0.x and the overall dtype in group bys only?

Namely:

- In non-group by context, the result is correct:
    ```py
    pd.Series([1,None, 2]).convert_dtypes(dtype_backend="pyarrow").var()
    0.5
    ```

- if we change the dtype to float, the result is correct:

    ```py
    (
      pd.DataFrame({"a": [1,1,1, 2], "b": [1,None, 2., 100.2]})
      .convert_dtypes(dtype_backend="pyarrow")
      .groupby("a", as_index=False).var()
    )
       a     b
    0  1   0.5
    1  2  <NA>
    ```

- Yet this is incorrect:
    ```py
     (
       pd.DataFrame({"a": [1,1,1, 2], "b": [1,None, 2., 100]})
       .convert_dtypes(dtype_backend="pyarrow")
       .groupby("a", as_index=False).var()
     )
        a     b
    0  1     0
    1  2  <NA>
    ```

@MarcoGorelli I would say that this is ok to xfail for now by checking pandas version and backend WDYT?

We could re-use std implementation, which curiously enough, does not behave in the same way, yet it fails to maintain the backend dtype for such pandas version:

```py
pd.DataFrame({"a": [1,1,1, 2], "b": [1,None, 2., 100]}).convert_dtypes(dtype_backend="pyarrow").groupby("a", as_index=False).std().dtypes
a    int64[pyarrow]
b           float64
dtype: object
```

while it should be 

```py
a    int64[pyarrow]
b    float64[pyarrow]
dtype: object
```

Reply:
yeah i'd say ok to xfail